interpreter/cling/lib/MetaProcessor/MetaSema.cpp

```diff
@@ -386,6 +386,12 @@ namespace cling {
       "\n"
       "   " << metaString << "help\t\t\t- Shows this information\n"
       "\n"
+      "   " << metaString << "help ClassName\t\t- Opens Doxygen online reference guide for ClassName\n"
```

Comment:
Please remove this here as that new help is not cling's but ROOT's, we need to move this into a new `InterpreterCallback`. We will do that in a separate PR, it's a bit of work...